packages/fuel-indexer-graphql/src/dynamic.rs

```diff
@@ -0,0 +1,457 @@
+use std::collections::{HashMap, HashSet};
+
+use async_graphql::{
+    dynamic::{
+        Enum, Field, FieldFuture, FieldValue, InputObject, InputValue, Object,
+        ResolverContext, Scalar, Schema as DynamicSchema,
+        SchemaBuilder as DynamicSchemaBuilder, TypeRef,
+    },
+    Request,
+};
+use async_graphql_parser::types::{BaseType, Type};
+use fuel_indexer_database::{queries, IndexerConnectionPool};
+use fuel_indexer_schema::db::tables::Schema;
+use lazy_static::lazy_static;
+use serde_json::Value;
+
+use crate::graphql::{GraphqlError, GraphqlQueryBuilder};
+
+lazy_static! {
```

Comment:
Can we include a rust-doc/comment mentioning these should always stay up-to-date with base.graphql ? (our base scalars). Also might help to keep this list sorted

Reply:
Yep, will do!